packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts

```diff
@@ -97,11 +98,11 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
             blur: {
                 beforeDispatch: () => {
                     if (this.editor) {
-                        this.applyFormatWithContentModel(
-                            this.editor,
-                            this.isCropMode,
-                            true /* shouldSelectImage */
-                        );
+                        // this.applyFormatWithContentModel(
+                        //     this.editor,
+                        //     this.isCropMode,
+                        //     true /* shouldSelectImage */
+                        // );
```

Comment:
Should we remove this?

Reply:
I was using to debug and forgot to uncomment it!